pkg/monitor/datapath_trace.go

```diff
@@ -89,13 +87,30 @@ func (tn *TraceNotifyV0) decodeTraceNotifyVersion0(data []byte) error {
 	return nil
 }
 
-// TraceNotifyV1 is the version 1 message format.
+// TraceNotifyV1 is the version 1 message format. This struct needs to be kept
+// in sync with the decodeTraceNotifyVersion1 func.
 type TraceNotifyV1 struct {
 	TraceNotifyV0
 	OrigIP types.IPv6
 	// data
 }
 
+// decodeTraceNotifyVersion1 decodes the trace notify message in 'data' into
+// the struct. This function needs to be kept in sync with the TraceNotifyV1
+// struct.
+func (tn *TraceNotifyV1) decodeTraceNotifyVersion1(data []byte) error {
+	if len(data) < traceNotifyV1Len {
+		return errors.New("unexpected length of data to decode into TraceNotifyV1")
```

Comment:
same, please print the `len(data)` in the error, sometimes it helps to understand the reason of the misconfiguration

Reply:
done